hack/build-local-images.py

```diff
@@ -102,23 +102,23 @@
         "tag": "latest",
         "directory": "recycler",
         "binaries": {
-            "openshift": "/usr/bin/openshift"
+            "oc": "/usr/bin/oc"
         },
         "files": {}
     },
     "docker-builder": {
         "tag": "latest",
         "directory": "builder/docker/docker-builder",
         "binaries": {
-            "openshift": "/usr/bin/openshift"
+            "oc": "/usr/bin/oc"
```

Comment:
@openshift/sig-developer-experience fyi the hack build local images script was broken for our docker-builder image.  It wasn't updating the right file in the image, so your changes would not be reflected.